containers/mariadb-local/test/testserver.sh

```diff
@@ -6,7 +6,7 @@ IMAGE="$1"  # Full image name with tag
 MYSQL_VERSION="$2"
 CONTAINER_NAME="testserver"
 HOSTPORT=33000
-MYTMPDIR=~/tmp/testserver-sh_$$
+MYTMPDIR=~"/tmp/testserver-sh_${SECONDS}_$$"
```

Comment:
If the goal is uniqueness, would it be better to use `$RANDOM` over `$SECONDS` here?